From 23f66f3288d0151bf849685517cd50674396f4c8 Mon Sep 17 00:00:00 2001
From: Daniel Ammann <daniel.ammann@bytesatwork.ch>
Date: Wed, 25 Jul 2018 14:10:38 +0200
Subject: [PATCH 1/4] arm: dts: am33xx: Add missing #phy-cells to
 ti,am335x-usb-phy

Fix warnings in ti dts files.

Inspired by mainline linux commit f0e11ff8ff6595cb49bc18929b525d35721b8383.
---
 arch/arm/dts/am33xx.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/dts/am33xx.dtsi b/arch/arm/dts/am33xx.dtsi
index 14caee7f0d..bb5248b80c 100644
--- a/arch/arm/dts/am33xx.dtsi
+++ b/arch/arm/dts/am33xx.dtsi
@@ -507,6 +507,7 @@
 				reg-names = "phy";
 				status = "disabled";
 				ti,ctrl_mod = <&usb_ctrl_mod>;
+				#phy-cells = <0>;
 			};
 
 			usb0: usb@47401000 {
@@ -555,6 +556,7 @@
 				reg-names = "phy";
 				status = "disabled";
 				ti,ctrl_mod = <&usb_ctrl_mod>;
+				#phy-cells = <0>;
 			};
 
 			usb1: usb@47401800 {
-- 
2.11.0

